### PR TITLE
HVX `F32-raddstoreexpminusmax` microkernels for Softmax

### PR DESCRIPTION
**Pull request by ejparkqc**
- Initial implementation and test added.
- xnnpack/intrinsics-polyfill.h has the horizontal sum code (Q6_f32_vrsum_Vsf) using vshuff and vadd.

### CODE REVIEW

File: src/f32-raddstoreexpminusmax/hvx-rr2-p5.c.in [hunk 1]
```diff
@@ -0,0 +1,200 @@
+// Copyright 2024 Google LLC
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+$assert BATCH_TILE % 4 == 0
```
Comment (fbarchard) [inline src/f32-raddstoreexpminusmax/hvx-rr2-p5.c.in:6]: multiple of 32 floats.
$assert BATCH_TILE % 32 == 0

File: src/xnnpack/intrinsics-polyfill.h [hunk 2]
```diff
@@ -317,4 +330,40 @@ HVX_Vector Q6_Vw_vmpyie_VwVh(HVX_Vector multiplier_lo, HVX_Vector multiplier_hi,
...
+    return *((float *) &vin);
+}
+
+static XNN_INTRINSIC
+HVX_Vector Q6_Vsf_vdiv_VsfVsf(HVX_Vector vin1, HVX_Vector vin2){
+    float* svin1 = (float *) &vin1;
+    float* svin2 = (float *) &vin2;
+
+    for(int i = 0; i < 32; i++)
+      svin1[i] = svin1[i] / svin2[i];
```
Comment (fbarchard) [inline src/xnnpack/intrinsics-polyfill.h:365]: suggest adding a todo to improve on this.   if div is not an option, perhaps try to make calling code use 'nr' - reciprocal.

**Review by fbarchard — APPROVED** [review]

File: src/xnnpack/intrinsics-polyfill.h (hunk 2 above)
Comment (ejparkqc) [reply src/xnnpack/intrinsics-polyfill.h:368]: Quick question - does XNNPACK has nr code already implemented or should we add? I added as a TODO for now, but definitely want to improve this.

**Comment by alankelly** [comment]
Thanks, importing.

**Pull request merged**